starfinder/db/models.py

```diff
@@ -295,25 +512,27 @@ class Race(db_engine.Model, ModelBase):
     description = db_engine.Column(db_engine.String(64), nullable=False)
     hit_points = db_engine.Column(db_engine.Integer(), nullable=False)
     race_type = db_engine.Column(db_engine.String(64), nullable=False)
-    physical_description = db_engine.Column(db_engine.String(64), nullable=False)
-    homeworld = db_engine.Column(db_engine.String(64), nullable=False)
-    society_and_alignment = db_engine.Column(db_engine.String(64), nullable=False)
-    relations = db_engine.Column(db_engine.String(64), nullable=False)
-    adventurers = db_engine.Column(db_engine.String(64), nullable=False)
-    names = db_engine.Column(db_engine.String(64), nullable=False)
+    physical_description = db_engine.Column(db_engine.String(64), nullable=True)
+    homeworld = db_engine.Column(db_engine.String(64), nullable=True)
+    society_and_alignment = db_engine.Column(db_engine.String(64), nullable=True)
+    relations = db_engine.Column(db_engine.String(64), nullable=True)
+    adventurers = db_engine.Column(db_engine.String(64), nullable=True)
+    names = db_engine.Column(db_engine.String(64), nullable=True)
 
     characters = orm.relationship('Character', backref='race')
     native_races = orm.relationship('NativeRace', backref='race')
     racial_traits = orm.relationship('RacialTrait', backref='race')
+    languages = orm.relationship('Language', backref='race')
 
 
 class RacialTrait(db_engine.Model, ModelBase, HasId):
+    id = db_engine.Column(db_engine.Integer(), primary_key=True)
     race_id = db_engine.Column(db_engine.ForeignKey("races.id"),
                                      nullable=False)
-    name = db_engine.Column(db_engine.String(64), nullable=False)
-    description = db_engine.Column(db_engine.String(64), nullable=False)
+    trait = db_engine.Column(db_engine.String(64), nullable=False)
```

Comment:
Let's call this one `name`. I'm imagining typing `racial_trait.trait`, and it doesn't seem very intuitive.